src/ir/enum_ty.rs

```diff
@@ -71,14 +71,18 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
+            let val = if is_signed {
+                cursor.enum_val_signed().map(EnumVariantValue::Signed)
+            } else {
+                if cursor.kind() == CXCursor_EnumConstantDecl {
```

Comment:
This looks like too much rightward drift. Why not meld this into the previous level and use `else if`?